community/unofficial-download.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Unofficial Downloads
+parent: Community
+permalink: /community/unofficial-downloads/
+---
+
+Requirements for being listed as an unofficial download of FreeTube:
+
+- The download must have been updated in the last month.
+- The download must be served via HTTPS.
```

Comment:
This can seem a bit weird if it's a package you install from the command line

Reply:
Hmm i understand, do you have a suggestion what to do with this?